unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/Material/MaterialHandler.cs

```diff
@@ -75,10 +90,20 @@ public void OnComponentModelUpdated(IParcelScene scene, IDCLEntity entity, PBMat
                 {
                     material = materialAsset.material
                 });
+                UniTask.RunOnThreadPool(async () =>
```

Comment:
I'd recommend that this kind of pieces of code include a comment, it is really hard to reason what it does and why

Reply:
Does this work on WebGL?
Anyways, I'd recommend you use the util class `TaskUtils.Run`